README.md

```diff
@@ -14,6 +14,6 @@ The web server will be available at ```http://127.0.0.1:10001/```
 ## Viewing xml
 
 To view the XML files: navigate to ```http://127.0.0.1:10001/XML/<XMLNAME.xml>```
-Full list of XML can be viewed [here](./XML).
+Full list of XML can be viewed [here](./project-files/XML).
```

Comment:
Please squash this commit to 191f3c32ac123a9d11a62aaf955620b087099c49, no need to have it separate.

Reply:
The history should be cleaner now. [https://github.com/JaanTaponen/DCC-XML-CSS-UI-demo/commits/validation-demo](https://github.com/JaanTaponen/DCC-XML-CSS-UI-demo/commits/validation-demo)